src/components/Footer.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Alert from './Alert'
+
+
+
+const Footer = () => {
+
+
+    return(
+        <>
+            <Alert />
+        </>
+    )
+
+}
+
+
+export default Footer
```

Comment:
Why is the `Alert` component placed inside the `Footer` component?

Reply:
Since the footer is gonna be on every page....then that's a good place to place it. 

Alert is gonna be an independent float feature...it's not necessarily linked to any component.

Reply:
`Navbar` is going to be on all the pages also, that's a better place to insert this component.